assets/src/state/searchPageState.ts

```diff
@@ -76,28 +78,37 @@ export const submitSearch = (): SubmitSearchAction => ({
   type: "SUBMIT_SEARCH",
 })
 
-interface ClearSearchAction {
-  type: "CLEAR_SEARCH"
+interface NewSearchSessionAction {
+  type: "NEW_SEARCH_SESSION"
+  payload: SelectedEntity | null
 }
-export const clearSearch = (): ClearSearchAction => ({
-  type: "CLEAR_SEARCH",
+export const newSearchSession = (
+  newSelection?: SelectedEntity | null
+): NewSearchSessionAction => ({
+  type: "NEW_SEARCH_SESSION",
+  payload: newSelection || null,
 })
 
-interface SelectVehicleAction {
-  type: "SELECT_SEARCH_VEHICLE"
-  payload: { vehicleId: VehicleId } | null
-}
-
```

Comment:
The SELECT_SEARCH_VEHICLE action was a specific case of SELECT_SEARCH_ENTITY, which resulted in some duplication. I opted to keep the `setSelectedVehicle` helper function but have it map to the "SELECT_SEARCH_ENTITY" instead. Though now I'm realizing `setSelectedVehicle` is unused, so unless there is feedback to keep it I'll remove that entirely!

Reply:
I'm fine with just getting rid of it!